src/main/java/org/apache/commons/collections4/IteratorUtils.java

```diff
@@ -123,7 +123,7 @@
     private static final String DEFAULT_TOSTRING_DELIMITER = ", ";
 
     /**
-     * IteratorUtils is not normally instantiated.
+     * {@code IteratorUtils} should not normally be instantiated.
```

Comment:
Hi @arturobernalg 
If you want to change the all the comments, I do not think we need to duplicate the class names in each comment, I would simply say:
```
/**
 * Don't allow instances.
 */
```

Reply:
OK, changed and rebased on 1 commit